app/javascript/packages/document-capture/components/location-collection-item.tsx

```diff
@@ -32,56 +32,64 @@ function LocationCollectionItem({
-            }}
-            type="submit"
-          >
-            {t('in_person_proofing.body.location.location_button')}
-          </Button>
-        </div>
-        <div>{streetAddress}</div>
-        <div>{formattedCityStateZip}</div>
-        {(weekdayHours || saturdayHours || sundayHours) && (
-          <h4>{t('in_person_proofing.body.location.retail_hours_heading')}</h4>
-        )}
-        {weekdayHours && (
-          <div>
-            {`${t('in_person_proofing.body.location.retail_hours_weekday')} ${weekdayHours}`}
+        <div className="grid-row">
+          <div className="grid-col-fill">
+            {numericDistance && (
+              <h3 className="usa-collection__heading margin-bottom-1">
+                {t('in_person_proofing.body.location.distance', {
+                  count: parseFloat(numericDistance),
+                })}
+              </h3>
+            )}
+            {!distance && <h3 className="usa-collection__heading margin-bottom-1">{name}</h3>}
+            <div>{streetAddress}</div>
+            <div>{formattedCityStateZip}</div>
+            {(weekdayHours || saturdayHours || sundayHours) && (
+              <h4>{t('in_person_proofing.body.location.retail_hours_heading')}</h4>
+            )}
+            {weekdayHours && (
+              <div>
+                {`${t('in_person_proofing.body.location.retail_hours_weekday')} ${weekdayHours}`}
+              </div>
+            )}
+            {saturdayHours && (
+              <div>
+                {`${t('in_person_proofing.body.location.retail_hours_sat')} ${saturdayHours}`}
+              </div>
+            )}
+            {sundayHours && (
+              <div>
+                {`${t('in_person_proofing.body.location.retail_hours_sun')} ${sundayHours}`}
+              </div>
+            )}
+            {phone && (
+              <div>
+                <h4>{t('in_person_proofing.body.location.contact_info_heading')}</h4>
+                <div>{`${t('in_person_proofing.body.location.phone')} ${phone}`}</div>
+              </div>
+            )}
+            <Button
+              id={`location_button_mobile_${selectId}`}
+              className="tablet:display-none margin-top-2 width-full heey"
+              onClick={(event) => handleSelect(event, selectId)}
+              type="submit"
+            >
+              {t('in_person_proofing.body.location.location_button')}
+            </Button>
           </div>
-        )}
-        {saturdayHours && (
-          <div>{`${t('in_person_proofing.body.location.retail_hours_sat')} ${saturdayHours}`}</div>
-        )}
-        {sundayHours && (
-          <div>{`${t('in_person_proofing.body.location.retail_hours_sun')} ${sundayHours}`}</div>
-        )}
-        {phone && (
-          <div>
-            <h4>{t('in_person_proofing.body.location.contact_info_heading')}</h4>
-            <div>{`${t('in_person_proofing.body.location.phone')} ${phone}`}</div>
+          <div className="grid-auto">
```

Comment:
I think the class name is `grid-col-auto` ([docs](https://designsystem.digital.gov/utilities/layout-grid/#auto-layout-columns)), although it seems to work just the same if the class is left out altogether.

```suggestion
          <div className="grid-col-auto">
```